app/src/main/res/layout/dialog_about.xml

```diff
@@ -56,7 +56,8 @@
             android:layout_width="fill_parent"
             android:layout_height="wrap_content"
             android:gravity="center_horizontal"
+            android:text="@string/dialog_about_others_summary"
```

Comment:
This shouldn't be necessary, the text is set in `PreferencesInformationAbout` and this would be overridden.